projects/wave-core/src/lib/operators/dialogs/source-operator-list/source-operator-list.component.html

```diff
@@ -72,4 +72,12 @@ <h4 mat-line>Country Selection</h4>
         <p mat-line>Select Country Borders</p>
     </mat-list-item>
 
+    <mat-list-item
```

Comment:
we should remove that addition to this file since it won't be there after the refactoring from the other PR

Reply:
merged with the other PR / master